### DISCUSSION TITLE
Extract javax.ws.rs.core.HttpHeaders

### DISCUSSION THREAD

**Discussion by KushagraBindal** [graphql-java/graphql-java#2352] [Q&A]
My requirement is to extract javax.ws.rs.core.HttpHeaders. Can someone please help in providing some help with this.

**Comment by KushagraBindal** [comment]
I have existing REST endpoints which are taking httpHeaders as the input but I am not able to get the same. Please help.

**Comment by bbakerman** [comment]
graphql-java is an engine for running graphql queries.  It knows nothing about HTTP.

This is a concern in layers beyond the graphql-java part
